### DISCUSSION TITLE
欢迎 | Hexo Theme Nexmoe

### DISCUSSION THREAD

**Discussion by giscus** [theme-nexmoe/hexo-docs#19] [General]
# 欢迎 | Hexo Theme Nexmoe

If you are a non-Chinese user, I recommend you to use Google Translate or deepl to translate this document.

https://docs.nexmoe.com/?giscus=1e678fffcb67d48b63be59fayKzvwF8lC78jim15yWC%2BQMXk75mwrUnrs2yxUuHoIiNKhdz99dcDE2svdBb8zSWJWlCMGMH0zB%2FeBTRGyai5FuhwHWZhnH4TlWXZksy%2FYeCkcVC%2BSusZFV4S%2BlM%3D

**Comment by nexmoe** [comment]
233
